### PR TITLE
PR for quantlab-feedstock

### PR DESCRIPTION
**Pull request by lab4quant**
Hello, conda-forge team

I'm submitting this PR for `quantlab-feedstock` repo creation.

[quantlab](https://github.com/quantlabio/quantlab) is forked from [jupyterlab](https://github.com/jupyterlab/jupyterlab), which focuses on quantitative finance area. It's still in very early phase.

Please review, thanks!

Regards
lab4quant

### CODE REVIEW

File: recipes/quantlab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/quantlab/meta.yaml]: Please delete all comments.

File: recipes/quantlab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - quantlab_launcher >=0.1.1
```
Comment (synapticarbors) [inline recipes/quantlab/meta.yaml]: You're going to need a feedstock for `quantlab_launcher` before this recipe will build. Please submit another PR for that project.

File: recipes/quantlab/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - notebook >=4.2
+
+test:
+  imports:
+    - quantlab
+
+about:
+  home: https://github.com/quantlabio/quantlab
+  license: BSD 3-Clause
```
Comment (synapticarbors) [inline recipes/quantlab/meta.yaml]: Please add `license_file: LICENSE`

File: recipes/quantlab/meta.yaml (hunk 2 above)
Comment (lab4quant) [inline recipes/quantlab/meta.yaml]: thank you, PR#3304 submitted for `quantlab_launcher`

File: recipes/quantlab/meta.yaml (hunk 1 above)
Comment (lab4quant) [inline recipes/quantlab/meta.yaml]: thank you, this is done and pushed to quantlabio/staged-receipes:master

File: recipes/quantlab/meta.yaml (hunk 3 above)
Comment (lab4quant) [inline recipes/quantlab/meta.yaml]: thank you, this is done and pushed to quantlabio/staged-receipes:master

**Comment by lab4quant** [comment]
Hello, conda-forge team

I've submitted PR#3304 for [quantlab_launcher-feedstock](https://github.com/conda-forge/quantlab_launcher-feedstock), which is merged.

For this PR:

* `license file` added in `meta.yaml`
* comments removed

The change was pushed to `quantlabio/staged-receipes:master`

Please kindly review again, thank you!

Regards
lab4quant

**Comment by synapticarbors** [comment]
It looks like maybe the `quantlab_launcher` feedstock might have not been fully completed when the tests ran. Can you push an empty commit to restart the CI tests? You might also be able to just close and then re-open this PR.

**Comment by lab4quant** [comment]
@synapticarbors , I pushed another commit, `circleci` and `travis-ci` passed
But `appveyor` failed:

```
Packages/dependencies missing in current win-32 or win-64 channels: 
  - quantlab 0.1.2 py27_0 -> quantlab_launcher >=0.1.1
  - quantlab_launcher >=0.1.1
Command exited with code 1
```

I ran this [query](https://anaconda.org/search?q=quantlab_launcher) in `anaconda cloud`, and found only `linux-64` and `osx-64` platforms

How can I fix this?

Thank you!

**Comment by synapticarbors** [comment]
I had someone restart the windows build so the binaries should now be there. Let's try kickstarting a new build in this PR again and hopefully everything should work and we'll be good to merge.

**Comment by lab4quant** [comment]
@synapticarbors , thank you!

I pushed another commit, all test passed!

Please kindly merge.
